bundle-size/.gitignore

```diff
@@ -1,6 +1,5 @@
-app.yaml
-
 # You can maintain multiple configuration files (for your local development and
 # for the production environment) by naming them <something>.<config-file-name>.
-*.app.yaml
-*.env
+/*.app.yaml
+/*.env
+!/redacted.env
```

Comment:
Should this file now include only `.env`, since there's no longer an `example.app.yaml`?

Reply:
You are correct. Done.